iModelCore/BeSQLite/PublicAPI/BeSQLite/BeSQLite.h

```diff
@@ -519,10 +519,9 @@ enum DbResult
     BE_SQLITE_ERROR_SchemaUpgradeRequired   = (BE_SQLITE_IOERR | 15 << 24), //!< The schemas found in the database need to be upgraded.
     BE_SQLITE_ERROR_SchemaTooNew            = (BE_SQLITE_IOERR | 16 << 24), //!< The schemas found in the database are too new, and the application needs to be upgraded.
     BE_SQLITE_ERROR_SchemaTooOld            = (BE_SQLITE_IOERR | 17 << 24), //!< The schemas found in the database are too old, and the DgnDb needs to be recreated after extensive data transformations ("teleported").
-    BE_SQLITE_ERROR_SchemaLockFailed        = (BE_SQLITE_IOERR | 18 << 24), //!< Error acquiring a lock on the schemas before upgrade.
+    BE_SQLITE_ERROR_DataTransformRequired   = (BE_SQLITE_IOERR | 18 << 24), //!< Schema update need to update data.

```

Comment:
You've renamed one error code and removed another. Do you intend to update code like that in BeSQLite.ts in itwinjs-core to reflect these changes? `DbResult` is `@public`.

Reply:
Those codes that was renamed or removed are not used anymore. Yes, I did that on itwin.js side changes.

Though i have a question, does this count as breaking change? Should i wait for 4.x?

Reply:
Please link to itwinjs-core PR that makes those changes in your PR description.
Yes, removing/renaming enum members is a breaking change.
IMO the "codes and locks" one can hang around unused for now.
Is your change in the semantics of the other one important enough to warrant renaming it?

Reply:
No, i can add a new one though the IOERR sub flags are all used so I renamed one as they are not used. I will link the PR in minute.

Reply:
Diff will be really bad but i did following
1. Sync BE_SQLITE_* with SQLITE_* errors and sort them. 
2. BE_SQLITE_ERROR_DataTransformRequired is assigned new value.

Made similar change to https://github.com/iTwin/itwinjs-core/pull/4934